src/pages/Account/FirebaseEmailHandler/EmailVerify.tsx

```diff
@@ -0,0 +1,145 @@
+import * as React from 'react';
+import Loading from 'shared/loading/Loading';
+import { auth } from 'utils/firebase';
+import { Button, Container, Fade, Row } from 'reactstrap';
+import { Link } from 'react-router-dom';
+import { FirebaseConsumer, FirebaseUserProps } from 'HOCs/FirebaseProvider';
+import {
+  CONTAINER_CLASSES,
+  CONTENT_CLASSES,
+  PRIMARY_BUTTON_CLASSES
+} from 'styled/custom.styled';
+import { VIEWPORT_CENTER_LAYOUT } from 'styled/sharedClasses/layout';
+
+interface State {
+  isSubmitting: boolean;
+  hasError: boolean;
+  showVerifyEmailSuccess: boolean;
+  errorMessage: string;
+}
+
+interface Props {
+  oobCode: string; //firebase code from email
+}
+
+export default class EmailVerify extends React.Component<Props> {
+  readonly state: State = {
+    isSubmitting: true,
+    hasError: false,
+    showVerifyEmailSuccess: false,
+    errorMessage: '',
+  };
+
+  componentDidMount() {
+    const oobCode = this.props.oobCode;
+
+    if (!oobCode) {
+      alert('Error, no oobCode defined.');
+    }
+
+    this.handleVerifyEmail(oobCode);
+  }
+
+  handleVerifyEmail(oobCode: string) {
+    auth
+      .applyActionCode(oobCode)
+      .then(() => {
+        this.setState({
+          showVerifyEmailSuccess: true,
+        });
+      })
+      .catch(err => {
+        console.error(err);
+        this.setState({
+          hasError: true,
+          errorMessage: 'There was an error verifying your email.',
+        });
+      })
+      .finally(() => this.setState({ isSubmitting: false }));
+  }
+
+  renderError() {
+    return (
+      <Container tag={Fade} className={CONTAINER_CLASSES}>
+        <h2 className={CONTENT_CLASSES.TITLE}>Sorry, there was an error</h2>
+        <p className={CONTENT_CLASSES.SUBTITLE}>{this.state.errorMessage}</p>
+        <Row>
+          <a target="_blank" href="https://support.beenest.com/">
```

Comment:
you can move the `button` to the `a` tag so the pointer is shown:
``<a href="..." className="btn btn-primary">Contact us</a>``

Reply:
👍